svthalia/concrexit
# Add foreignkey between photos and event

Closes #1363 



### Summary
Added an optional foreign key between photo album and event. In a photo album you can click on event to go to the linked event, and on an event page you can click on the linked albums to go to one of the corresponding photo albums.
Secondly there is also a script to link the old events and albums. This is done by making a csv file which you can then edit and is read to link the albums and events you put in the same row.

### How to test
Test the feature:
1. Go to photo album admin page
2. Add an event

3. Go to the album page 
4. Click on the 'event' button
5. On the event page, click on the 'album' button

Test the script:

1. Make some events and albums on the same day
2. Run the linking script link_album_events
3. Edit the csv file
4. Check if the events and albums are properly linked
